packages/react/__tests__/__snapshots__/exports.ts.snap

```diff
@@ -5041,9 +5041,6 @@ Object {
       "ariaValuetext": Object {
         "type": "string",
       },
-      "asChild": Object {
```

Comment:
We don't support these props in Expander, so it should be ok to remove them

Reply:
Did this show up in our docs somewhere?

Reply:
This never showed in our docs: https://ui.docs.amplify.aws/react/components/expander?tab=props